### ISSUE TITLE
FYI:  Tag Assistant and pageVisibilityTracker Issue

### ISSUE THREAD

**Issue by agustinv** [googleanalytics/autotrack#204]
When setting sendInitialPageview to true using PageVisibilityTracker I can correctly confirm that page view is correctly being sent to GA by looking at the network calls to google-analytics:

eg payload includes v=1&_v=j64&a=31267556&t=pageview ...

However Tag assistant doesnt report it upon landing ie first page load on site.

https://ga-dev-tools.appspot.com/tag-assistant/

You can probably close this issue since its not IMO an autotrack issue but adding it here in case someone is running into the same situation.

**Comment by philipwalton** [comment]
Yeah, this is because Tag Assistant doesn't know about [`beacon`](https://developers.google.com/analytics/devguides/collection/analyticsjs/sending-hits#specifying_different_transport_mechanisms), which autotrack always uses when available.

I filed a bug about this internally a long time ago but haven't seen any traction. I'll re-ping that thread and mention external users are complaining about it. Hopefully that'll help.
